code/modules/surgery/rig_removal.dm

```diff
@@ -0,0 +1,58 @@
+//Procedures in this file: Unsealing a Rig.
```

Comment:
The problem i would have with no NODROP is a bit that the player themselfs can just take it off, and they unseal the whoole rigsuit in no time, unless you can get some sort of timer in, like putting the backpack on. I wouldn't know how to do that, but that feature would also be good.